exercises/colorsquare.js

```diff
@@ -82,6 +103,14 @@ class ColorSquare{
 		return the dom element that was generated. 
 	*/
 	render(){
-
+        var newElement = $("<div>",{
+            "class": this.domElementClass,
+            on: {
+                click: this.handleClick
+            },
+            "background-color": this.colorsArray[this.colorIndex]
```

Comment:
You also could have set the element  to your property below and then called your changeColor function